augly/tests/image_tests/functional_unit_test.py

```diff
@@ -21,6 +21,9 @@ def test_blur(self):
 
     def test_brightness(self):
         self.evaluate_function(imaugs.brightness)
+    
+    def test_skew(self):
```

Comment:
Remove this test. You have put the functional test twice. Also add a test in the `transforms_unit_test.py` function. Also regenerate the output file `test_skew.png` because this unit test is now failing.